airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -9020,7 +9020,7 @@
         oauthFlowOutputParameters:
         - - "token"
         - - "key"
-- dockerImage: "airbyte/source-twilio:0.1.4"
+- dockerImage: "airbyte/source-twilio:0.1.5"
```

Comment:
Please do not bump the version here as we have an automation that will take care of this 🙏